packages/cli/test/dev/integration-1.test.ts

```diff
@@ -207,8 +207,8 @@ test('[vercel dev] should handle config errors thrown in edge functions', async
     expect(await res.text()).toMatch(
       /<strong>500<\/strong>: INTERNAL_SERVER_ERROR/g
     );
-    expect(stderr).toMatch(
-      /Invalid function runtime "invalid-runtime-value" for "api\/edge-error-config.js". Valid runtimes are: \["edge","experimental-edge"\]/g
+    expect(stderr).toContain(
+      'unsupported "runtime" value in `config`: "invalid-runtime-value" (must be one of: ["edge","experimental-edge"]). Learn more: https://vercel.link/creating-edge-functions'
```

Comment:
Looks like the error message no longer tells you which file is problematic

Reply:
Oh the file name is at the start of the error message so maybe check for that too?

Reply:
Done! [`f240ba5` (#9851)](https://github.com/vercel/vercel/pull/9851/commits/f240ba56d080f340bfd2ef7d8d6d902656a0a02f)